common/client-libs/gateway-client/src/client.rs

```diff
@@ -540,6 +540,9 @@ impl<C, St> GatewayClient<C, St> {
                 self.bandwidth_remaining = bandwidth_remaining;
                 self.negotiated_protocol = protocol_version;
                 log::debug!("authenticated: {status}, bandwidth remaining: {bandwidth_remaining}");
+                self.task_client.send_status_msg(Box::new(
```

Comment:
just making sure: you're only sending this message on authentication, but I guess after we merge ecash, this will be `Arc<AtomicI64>` so you will be able to query it to your heart's content

Reply:
yep that's the idea